setup.cfg

```diff
@@ -0,0 +1,51 @@
+[metadata]
+version = attr: keras_nlp.__version__
+
+[tool:pytest]
+filterwarnings =
+    error
+    ignore::DeprecationWarning
+    ignore::ImportWarning
+    ignore::RuntimeWarning
+    ignore::PendingDeprecationWarning
+    ignore::FutureWarning
+
+addopts=-vv
+
+# Do not run tests in the build folder
+norecursedirs= build
+
+[isort]
+known_first_party = keras_nlp,tests
+default_section = THIRDPARTY
+line_length = 80
+
+[coverage:report]
```

Comment:
We may need to exclude the "*_test.py" files if we put them next to the source files. I haven't changed it yet.

Reply:
Good call. `omit = *_test.py` seemed to do the trick.